src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Text_Format.txt

```diff
@@ -767,4 +767,16 @@ Error({Kind:ErrorKind.BadLanguageCode})
 Error({Kind:ErrorKind.BadLanguageCode})
 
 >> Text(1234.5678, "[$-fr-FR # ##]0,00", "vi-VI")
-Error({Kind:ErrorKind.BadLanguageCode})
\ No newline at end of file
+Error({Kind:ErrorKind.BadLanguageCode})
+
+>> Text(ParseJSON("""\u202F"""), "##")
+Error({Kind:ErrorKind.InvalidArgument})
+
+>> Text("Hello", If(1 < 0, "d", Blank()))
+""
+
+>> Text("Hello", If(1 < 0, "d", ""))
+"Hello"
+
+>> Text("Hello", If(1 > 0, "d", ""))
+Error({Kind:ErrorKind.InvalidArgument})
```

Comment:
Wouldn't those be a compile-time error if PowerFxV1 is enabled?

Reply:
I see, i thought i will let it check at runtime so if it pass empty string format then the case still valid.
I will update logic to make it fail at compile time.